File: discover_ruby/github_most_popular.rb
require 'httpclient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 6309c44b1fa175e71452a2b7e390792d807b08dc'
}

clnt = HTTPClient.new
url = 'https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc'
puts clnt.get_content(url, nil, extheaders)

File: discover_more_python/github_most_popular.py
from urllib2 import Request, urlopen, URLError

request = Request('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc')

try:
	response = urlopen(request)
	github_data = response.read()
	print github_data
except URLError, e:
    print 'Error', e
